smithy-diff/src/test/resources/software/amazon/smithy/diff/evaluators/configurable/configurable-b.smithy

```diff
@@ -0,0 +1,54 @@
+$version: "2.0"
+
+metadata diffEvaluators = [
```

Comment:
Maybe this should emulate validators: https://smithy.io/2.0/spec/model-validation.html#validators. So more like:

```
{
    id: "AddedInternalOperation"
    name: "EmitEachMatch|EmitIfNoneMatch|EmitIfAnyMatch|..."
    severity: "NOTE"
    configuration: {
        appliesTo: "AddedShapes|RemovedShapes|ChangedShapes"
        selector: "operation [trait|internal]"
    }
}
```

This keeps the door open for future configurable diff evaluators, and maybe even the ability to configure built-in evaluators. This could also, if we really look into the future, potentially cover metadata diffs and trait diffs too.